From b18a23495fd4492081a61373fc3ef9f5c59e094a Mon Sep 17 00:00:00 2001
From: Matthew Wilcox <willy@linux.intel.com>
Date: Fri, 30 Apr 2010 13:11:28 -0600
Subject: USB: Add definition for the Pipe Usage descriptor
Patch-mainline: v2.6.35
Git-commit: b18a23495fd4492081a61373fc3ef9f5c59e094a

The Pipe Usage descriptor is needed for USB Attached SCSI

Signed-off-by: Matthew Wilcox <willy@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/ch9.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/include/linux/usb/ch9.h
+++ b/include/linux/usb/ch9.h
@@ -191,6 +191,8 @@ struct usb_ctrlrequest {
 #define USB_DT_WIRE_ADAPTER		0x21
 #define USB_DT_RPIPE			0x22
 #define USB_DT_CS_RADIO_CONTROL		0x23
+/* From the T10 UAS specification */
+#define USB_DT_PIPE_USAGE		0x24
 /* From the USB 3.0 spec */
 #define	USB_DT_SS_ENDPOINT_COMP		0x30
 
